From bd357b34f2e699b324a5248f9c82d55f75e98152 Mon Sep 17 00:00:00 2001
From: JPyke3 <github@pyk.ee>
Date: Mon, 17 Aug 2026 07:45:07 +1000
Subject: [PATCH 1/2] dt-bindings: input: goodix: Document input device label

Document the optional label property used to provide a descriptive name for
the input device.

Signed-off-by: JPyke3 <github@pyk.ee>
---
 .../devicetree/bindings/input/touchscreen/goodix.yaml         | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml b/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
index 427ca64..e93222b 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
@@ -48,6 +48,10 @@ properties:
   interrupts:
     maxItems: 1
 
+  label:
+    description: Descriptive name of the input device
+    $ref: /schemas/types.yaml#/definitions/string
+
   irq-gpios:
     description: GPIO pin used for IRQ input. Additionally, this line is
       sampled by the device on reset deassertion to select the I2C client
-- 
2.55.0

